src/Components/TodoListItem/TodoListItem.jsx

```diff
@@ -0,0 +1,111 @@
+import styles from "./TodoListItem.module.css";
+import { FaTrash } from "react-icons/fa";
+import { FaEdit } from "react-icons/fa";
+import { FaSave } from "react-icons/fa";
+import PropTypes from "prop-types";
+import { useState } from "react";
+
+function TodoListItem({ title, onRemoveTodo, id, createdTime, onEditTodo , isCompleted}) {
+  const [isChecked, setIsChecked] = useState(isCompleted);
+  const [isEditing, setIsEditing] = useState(false);
+  const [updatedTodoTitle, setUpdatedTodoTitle] = useState("");
```

Comment:
Can you pre-populate input with the current data, what do you think? 

https://github.com/user-attachments/assets/c6bea5c5-593a-4642-ab40-49e4418ff685

Reply:
fixed!